### DISCUSSION TITLE
4.0 Release schedule

### DISCUSSION THREAD

**Discussion by ipichris** [smallrye/smallrye-open-api#2000] [General]
Is there a release schedule for the 4.0 release?
Will there be another alpha or release candidate? The last alpha is almost 2 months old and doesn't contain the OpenAPI 3.1 features I'm looking forward to.

Thanks for the great work on this project!

**Comment by MikeEdgar** [comment]
Hi @ipichris , it will probably be next week some time. Issue #1987 still needs to be resolved first. Which features are you looking for? The last alpha should be feature-complete as it was used to verify the MP OpenAPI 4.0 specification release.

**Comment by ipichris** [comment]
Thanks, I will wait for the release then
